org-docs/CHARTER.md

```diff
@@ -68,6 +68,14 @@ In order to be eligible to be a Organization project, a project must:
 
 Amendments to this charter, the [antitrust policy](./ANTITRUST.md), the [trademark policy](./TRADEMARKS.md), or the [code of conduct](./CODE-OF-CONDUCT.md) may only be made with at least a 3/4 affirmative vote of the Steering Committee.
 
+**9.1 Officers and Duties**. Every six months, the Steering Committee will conduct a vote for the following officers:
+
+* **Chair**. The Chair's main duty is to chair Steering Committee meetings. The Chair gives overall leadership to the Steering Committee, and should ensure that the Steering Committee members understand the roles and responsibilities, and debate its mission. The Chair also ensures that Steering Committee meetings are effectively run, and that decisions are made and properly recorded.
+
+* **Vice-Chair**. In the event of the Chair being absent, the Vice-Chair must undertake all chairing duties and ensure the objectives of Steering Committee meetings are met. The Vice-Chair may deputize for the Chair in all other duties as required.
+
+* **Secretary**. The Secretary ensures that Steering Committee meetings are run smoothly, records detailed minutes that capture all key points of Steering Committee meetings, and distributes minutes to all Steering Committee members within a week of the meeting. The Secretary also schedules and prepares invitations for Steering Committee meetings, and creates agendas according to the organization’s bylaws and distributes them to Steering Committee members, along with any supporting documentation.
```

Comment:
"and creates agendas according to the organization’s bylaws"

Doesn't the agenda come from the chair?

Reply:
> "and creates agendas according to the organization’s bylaws"
> 
> Doesn't the agenda come from the chair?

These are the rules we voted on.  @kevincol  please confirm